src/textual/dom.py

```diff
@@ -1139,12 +1140,12 @@ def query_one(
 
         return query.only_one() if expect_type is None else query.only_one(expect_type)
 
-    def set_styles(self, css: str | None = None, **update_styles) -> Self:
+    def set_styles(self, css: str | None = None, **update_styles: Any) -> Self:
         """Set custom styles on this object.
 
         Args:
             css: Styles in CSS format.
-            **update_styles: Keyword arguments map style names on to style.
+            update_styles: Keyword arguments map style names onto style values.
```

Comment:
Griffe expects to find `*` and `**` in front of parameter names in the docstring too. Aren't you getting new warnings without them :thinking: ?